bee-message/Cargo.toml

```diff
@@ -37,3 +37,4 @@ num_cpus = { version = "1.13.0", default-features = false }
 [features]
 serde1 = [ "serde", "serde-big-array" ]
 std = [ ]
+cpt2 = [ ]
```

Comment:
Put this first (alphanumeric order)

Reply:
[814cf38](https://github.com/iotaledger/bee/pull/1075/commits/814cf389b7ebe5590fe300435122b485b918c34e)